pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -173,14 +173,18 @@ wxWindowID SwapOrReset_Iso(wxWindow* owner, IScopedCoreThread& core_control, con
 		core_control.DisallowResume();
 		wxDialogWithHelpers dialog(owner, _("Confirm ISO image change"));
 
-		dialog += dialog.Heading(descpart1);
-		dialog += dialog.GetCharHeight();
+		dialog += dialog.Heading(descpart1); // New lines already applied
 		dialog += dialog.Text(isoFilename);
 		dialog += dialog.GetCharHeight();
 		dialog += dialog.Heading(_("Do you want to swap discs or boot the new image (via system reset)?"));
 
-		result = pxIssueConfirmation(dialog, MsgButtons().Reset().Cancel().Custom(_("Swap Disc"), "swap"));
-		if (result == wxID_CANCEL)
+#ifndef DISABLE_RECORDING
+		if (g_InputRecording.IsActive() && g_InputRecording.GetInputRecordingData().FromSaveState())
+			dialog += dialog.Text(_("\n(Warning: The savestate accompanying the active input recording\nmay not be compatible with the new source)"));
+#endif
+
+		result = pxIssueConfirmation( dialog, MsgButtons().Reset().Cancel().Custom(_("Swap Disc"), "swap"));
+		if( result == wxID_CANCEL )
```

Comment:
Formatting.

Reply:
Clarification on this one?
Referencing the empty line or something else?

Reply:
ohhhhhhh